etc/base.yaml

```diff
@@ -178,17 +178,40 @@ postgresql:
         - name: POSTGRES_MULTIPLE_DATABASES
           value: restsourceauthorizer,appconfig,kratos # ... and these databases will be created
 
```

Comment:
Also maybe add a note to the postgresql configuration above, mentioning that it is deprecated.

Reply:
I added deprecation warnings as well as corrected comments around database deployments